scipy/special/xsf/config.h

```diff
@@ -144,7 +146,7 @@ XSF_HOST_DEVICE void swap(T &a, T &b) {
 }
 
 // Reimplement std::clamp until it's available in CuPy
-template<typename T>
+template <typename T>
```

Comment:
An unrelated remark: Line 151 seems to contain a typo — if `v > hi`, `hi` should be returned.

Reply:
Thanks for pointing out the typo. I guess `xsf::cephes::kolmogorov` isn't tested well enough. It's the only thing using `std::clamp`. I'll make a new PR to fix that.